packages/eslint-plugin-pf-codemods/src/rules/v6/mastheadNameChanges/masthead-name-changes.md

```diff
@@ -0,0 +1,18 @@
+### masthead-name-changes [(#10809)](https://github.com/patternfly/patternfly-react/pull/10809)
+
+The old `MastheadBrand` component has been renamed to `MastheadLogo`, and the old `MastheadMain` has been renamed to `MastheadBrand`.
```

Comment:
I think the message might be confusing for some consumers (or at least the messages given in the codemod output "MastheadMain has been renamed to MastheadBrand"), I like using the word "old" here, which makes more sense.

I was confused at the first place why are we renaming these components, when they will still stay in the new Masthead.

Maybe we can add a sentence here, like "`MastheadMain will now wrap both MastheadToggle and MastheadBrand within itself.`".

I know there will probably be a second codemod for wrapping the components in MastheadMain and the message there will better describe the whole change, but still this renaming which is not really renaming might be confusing (but probably not many consumers read the messages, I guess they just run the fix, so I am ok with keeping it as is).